src/System.Net.Requests/pkg/System.Net.Requests.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.Net.Requests.csproj">
-      <SupportedFramework>net46;netcore50;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
+      <SupportedFramework>net463;netcoreapp1.1;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
> Does it mean that it is supported?

Yes it is supported.  We make sure you never drop support for past frameworks.  We look at your last shipped package, find out what frameworks it supported and ensure that the new package supports all of those at the same or higher API version.  We also automatically include ref and impl from the last package in the new one if there is no live build of that asset.